### PR TITLE
UI mapping in Snap Pixel, Reddit Pixel and Google ads

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

* Revamped docs
* Added UI mapping support in Snap Pixel, Reddit Pixel and Google ads

### CODE REVIEW

**Review by yashasvibajpai — COMMENTED** [review]
We didn't add the screenshots of the mapping support that we now have fro mthe dashbaord?

**Comment by ishachopra0401** [comment]
@yashasvibajpai We are standardizing all the documents this way by adding screenshots only where they will help a reader to understand/relate more. Currently, screenshots occupy a lot of space in the docs and are practically a replica of the dashboard.

In these docs as well, the settings are straightforward forward and their description is listed in the same order as they are in the dashboard.

LMK if you think otherwise.

**Comment by yashasvibajpai** [comment]
> @yashasvibajpai We are standardizing all the documents this way by adding screenshots only where they will help a reader to understand/relate more. Currently, screenshots occupy a lot of space in the docs and are practically a replica of the dashboard.
> 
> In these docs as well, the settings are straightforward forward and their description is listed in the same order as they are in the dashboard.
> 
> LMK if you think otherwise.

Got it. Yes, I think this would be suitable.
This mapping support can be mentioned in connection settings. so there is no need for screenshot.

Looks good!

**Review by yashasvibajpai — APPROVED** [review]

File: docs/destinations/streaming-destinations/reddit-pixel.mdx [hunk 1]
```diff
@@ -57,41 +51,44 @@ rudderanalytics.page()
...
+A sample `track` call is shown below:
 
 ```javascript
 rudderanalytics.track("Product Added")
 ```
 
-RudderStack transforms the following events to Reddit Pixel's Standard Events:
+### Property mappings
+
+The following table lists the event mapping from RudderStack to Reddit Pixel's Standard Events:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/reddit-pixel.mdx:64]:
```suggestion
The following table lists the event mapping from RudderStack to Reddit Pixel's standard events:
```

File: docs/destinations/streaming-destinations/reddit-pixel.mdx [hunk 2]
```diff
@@ -57,41 +51,44 @@ rudderanalytics.page()
...
 
-| **RudderStack Event Name**  | **Reddit Pixel Standard Event** |
+| RudderStack event  | Reddit Pixel standard event |
 | :-------------------------- | :------------------------------ |
 | `Product Added`             | `Add to Cart`                   |
 | `Product Added to Wishlist` | `Add to Wishlist`               |
 | `Order Completed`           | `Purchase`                      |
 
-In addition, RudderStack also supports mapping the following custom events:
+The following table lists the custom events mapping from RudderStack to Reddit Pixel's Standard Events:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/reddit-pixel.mdx:72]:
```suggestion
The following table lists the custom events mapping from RudderStack to Reddit Pixel's standard events:
```

File: docs/destinations/streaming-destinations/g-ads-gtag.mdx [hunk 3]
```diff
@@ -9,85 +9,53 @@ RudderStack's Google Ads integration leverages a global site tag - `gtag.js` - a
...
+- **Mapping to trigger the Google Ads events for the respective events set here**: Use this setting to map custom event names to the standard Google Ads events.
+- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Google Ads. Refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide for more information.
+- **Configure Page Load Conversions**: For `page` call, you can configure the page load conversions for multiple instances. 
+  - **Conversion Label**: Enter the conversion label from Google Ads. 
+  - **Name**: Enter the name of the `page` event to be sent.
+- **Click Event Conversion**: For `track` call, you can configure **Click Event Conversion**. 
+  - **Conversion Label**: Enter your Google Ads conversion label. 
+  - **Name**: Enter the name of the `track` event to be sent.
+- **Send Page View**: Enable this setting to configure Google Ads to automatically send your `page` events.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/g-ads-gtag.mdx:41]:
```suggestion
- **Send Page View**: Enable this setting to automatically send your `page` events to Google Ads.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/g-ads-gtag.mdx:41]: The current description has some scope for confusion. Tweaked for more clarity.

File: docs/destinations/streaming-destinations/g-ads-gtag.mdx [hunk 4]
```diff
@@ -9,85 +9,53 @@ RudderStack's Google Ads integration leverages a global site tag - `gtag.js` - a
...
-
-<div class="infoBlock">
- 
-<strong>Dynamic Remarketing</strong> lets you tailor your ads based on your users' previous interactions with your website. For more information on this feature, refer to the official Google Ads <a href="https://support.google.com/google-ads/answer/7305793">documentation</a>.
-</div>
- 
-- As this is a device mode-only integration, the **Use device-mode to send events** option will always be enabled.
-
-- Finally, click on **Next** to finish the configuration. Google Ads will now be added and enabled as a destination in RudderStack.
+- **Disable Ad Personalization**: Enable this setting to programmatically disable ad personalization. Refer to the [Google Ads documentation](https://developers.google.com/analytics/devguides/collection/gtagjs/display-features#disable_advertising_personalization) for more information on disabling the advertising personalization feature.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/g-ads-gtag.mdx:48]:
```suggestion
- **Disable Ad Personalization**: Enable this setting to programmatically disable ad personalization. Refer to the [Google Ads documentation](https://developers.google.com/analytics/devguides/collection/gtagjs/display-features#disable_advertising_personalization) for more information.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/g-ads-gtag.mdx:48]: Feels repetitive, WDYT?

File: docs/destinations/streaming-destinations/g-ads-gtag.mdx [hunk 5]
```diff
@@ -9,85 +9,53 @@ RudderStack's Google Ads integration leverages a global site tag - `gtag.js` - a
...
+- **Use device mode to send events**: As this is a web device mode-only destination, this setting is enabled by default and cannot be disabled.
+- **Dynamic Remarketing**: Enable this setting to allow RudderStack to leverage Google Ads' Dynamic Remarketing feature for event tracking. Dynamic Remarketing lets you tailor your ads based on your users' previous interactions with your website. Refer to the [Google Ads documentation](https://support.google.com/google-ads/answer/7305793) for more information on this feature.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/g-ads-gtag.mdx:50]:
```suggestion
- **Dynamic Remarketing**: Enable this setting to allow RudderStack to leverage Google Ads' Dynamic Remarketing feature for event tracking. [Dynamic Remarketing](https://support.google.com/google-ads/answer/7305793) lets you tailor your ads based on your users' previous interactions with your website.
```

**Review by ameypv-rudder — APPROVED** [review]
Made some minor suggestions - LGTM otherwise.

**Pull request merged**